docs/DeveloperGuide.adoc

```diff
@@ -798,17 +802,65 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+|`* * *` |user |undo a command |easily revert to a previous state
+
+|`* * *` |user |redo a command |redo a command that was undone
+
+|`* * *` |user |select a person from the previously retrieved list |view more details about the particular customer
+
+|`* * *` |user |view a customer's address on Google Maps |visualize the location of the customer
 
-|`* * *` |user |add a new person |
+|`* * *` |user |back up my list |create backup copies of the current data in case of data corruption
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |user |list all my runners |get an overview of all subordinates
+
+|`* * *` |user |schedule collection of payment |plan  meetings with customers and facilitate timely collection of payments
+
+|`* * *` |user |schedule a house visit |send a friendly, personal reminder to a customer regarding payments
+
+|`* * *` |user |add a customer with partial information |add a customer entry without requiring all the details
+
+|`* * *` |user |easily make ad hoc calculations |perform simple payment calculations without disrupting my workflow
+
+|`* * *` |user |view customer or transaction statistics |make informed decisions or get relevant information about customers or transactions
+
+|`* * *` |user |rate my customers |know who the most reliable customers are
+
+|`* * *` |user |filter customers by area |make operations, collections and visitations more time and cost efficient
+
+|`* * *` |user |hide my application quickly |prevent unwanted eyes from confidential data and the activities of the syndicate
+
+|`* * *` |user |view information about a customer's collateral (family members, relatives, property)  |nalyse possible courses of action against recalcitrant customers who do not pay up
+
+|`* * *` |user |view a compact status view of customers using small symbols or pictorial representations |see general status information about customers at a glance
+
+|`* * *` |user |view different configurations of one or more routes, all together encompassing a given set of customers' locations |find the optimal route in terms of cost, time and effort that can be taken to collect from a given set of customers
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* *` |user |export data of customers and runners |view the data in another format or on another platform
+
+|`* *` |user |view a route encompassing a given set of customers' locations |access the cost, time and effort involved in collecting from a set of customers
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
+|`*` |user |export my current view to another format |view specific information in another format or print out a hardcopy and pass the information to subordinates
```

Comment:
Update MSS accordingly. I believe you do have new commands which have different workflow.